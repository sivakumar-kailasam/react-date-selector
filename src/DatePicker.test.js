import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import DatePicker from './DatePicker'

it('renders without crashing', () => {
  const tree = renderer.create(<DatePicker />).toJSON()
  expect(tree).toMatchSnapshot()
  expect(tree).toBeTruthy()
})
