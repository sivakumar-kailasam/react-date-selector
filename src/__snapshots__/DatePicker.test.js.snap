// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
.c0 {
  text-align: center;
}

.c1 {
  background-color: #222;
  height: 150px;
  padding: 20px;
  color: white;
}

.c2 {
  -webkit-animation: iVXCSc infinite 20s linear;
  animation: iVXCSc infinite 20s linear;
  height: 80px;
}

.c3 {
  font-size: 1.5em;
}

.c4 {
  font-size: large;
}

<div
  className="c0"
>
  <header
    className="c1"
  >
    <img
      alt="logo"
      className="c2"
      src="logo.svg"
    />
    <h1
      className="c3"
    >
      Welcome to React
    </h1>
  </header>
  <p
    className="c4"
  >
    To get started, edit 
    <code>
      src/App.js
    </code>
     and save to reload.
  </p>
</div>
`;
